src/javascripts/components/mobile-navigation.js

```diff
@@ -1,89 +1,112 @@
-import $ from 'jquery'
-
-var MobileNav = {
-  $mobileNav: $('.js-app-mobile-nav'),
-  $mobileNavToggler: $('.js-app-mobile-nav-toggler'),
-  mobileNavActiveClass: 'app-mobile-nav--active',
-  mobileNavTogglerActiveClass: 'app-header-mobile-nav-toggler--active',
-  $subNav: $('.js-app-mobile-nav-subnav'),
-  $subNavToggler: $('.js-mobile-nav-subnav-toggler'),
-  subNavActiveClass: 'app-mobile-nav__subnav--active',
-  subNavTogglerActiveClass: 'app-mobile-nav__subnav-toggler--active',
-
-  bindUIEvents: function () {
-    MobileNav.$mobileNavToggler.on('click', function (e) {
-      if (MobileNav.$mobileNav.hasClass(MobileNav.mobileNavActiveClass)) {
-        MobileNav.$mobileNav.removeClass(MobileNav.mobileNavActiveClass)
-        MobileNav.$mobileNav.attr('aria-hidden', 'true')
-
-        MobileNav.$mobileNavToggler.removeClass(MobileNav.mobileNavTogglerActiveClass)
-        MobileNav.$mobileNavToggler.attr('aria-expanded', 'false')
-      } else {
-        MobileNav.$mobileNav.addClass(MobileNav.mobileNavActiveClass)
-        MobileNav.$mobileNav.attr('aria-hidden', 'false')
-
-        MobileNav.$mobileNavToggler.attr('aria-expanded', 'true')
-        MobileNav.$mobileNavToggler.addClass(MobileNav.mobileNavTogglerActiveClass)
-      }
-    })
+import 'govuk-frontend/vendor/polyfills/Element/prototype/classList'
+
+import common from 'govuk-frontend/common'
+var nodeListForEach = common.nodeListForEach
+
+var mobileNavActiveClass = 'app-mobile-nav--active'
```

Comment:
Given we're in a file called mobile navigation, I'm wondering if it's worth removing some of the 'mobile' and 'mobile nav' prefixes throughout this file, and just using e.g. `activeClass`, `togglerActiveClass`, `$toggler` – just think it might make the code a little easier to follow.

Entirely appreciate however that this is just maintaining the existing variable names, and this in no way blocking – just a thought.